plugins/support_cc.lua

```diff
@@ -0,0 +1,54 @@
+-- mod-version:3 --lite-xl 2.1
+
+local core = require "core"
+local config = require "core.config"
+local common = require "core.common"
+
+local cc = common.merge({
+  backend = "gcc"
```

Comment:
Should we really differenciate between gcc and clang? clang is a drop in solution in most cases.